src/test/resources/log4j2.properties

```diff
@@ -1,18 +1,33 @@
-status = debug
-name = PropertiesConfig
-packages = org.folio.okapi.common.logging
-
-filters = threshold
-
-filter.threshold.type = ThresholdFilter
-filter.threshold.level = debug
-
 appenders = console
 
 appender.console.type = Console
 appender.console.name = STDOUT
-appender.console.layout.type = PatternLayout
+appender.console.layout.type = JSONLayout
+appender.console.layout.compact = true
+appender.console.layout.eventEol = true
+appender.console.layout.stacktraceAsString = true
+
+packages = org.folio.okapi.common.logging
+appender.console.layout.requestId.type = KeyValuePair
+appender.console.layout.requestId.key = requestId
+appender.console.layout.requestId.value = $${FolioLoggingContext:requestid}
+
+appender.console.layout.tenantId.type = KeyValuePair
+appender.console.layout.tenantId.key = tenantId
+appender.console.layout.tenantId.value = $${FolioLoggingContext:tenantid}
+
+appender.console.layout.userId.type = KeyValuePair
+appender.console.layout.userId.key = userId
+appender.console.layout.userId.value = $${FolioLoggingContext:userid}
+
+appender.console.layout.moduleId.type = KeyValuePair
+appender.console.layout.moduleId.key = moduleId
+appender.console.layout.moduleId.value = $${FolioLoggingContext:moduleid}
 
 rootLogger.level = debug
-rootLogger.appenderRefs = info
+rootLogger.appenderRefs = debug
 rootLogger.appenderRef.stdout.ref = STDOUT
+
+logger.netty.name = io.netty
```

Comment:
Why do you add netty properties?

Reply:
It is like misunderstanding. I have removed them. Thanks for CR.